Dizzy/UI Components/CommentsTextField.swift

```diff
@@ -40,17 +40,15 @@ class CommentsTextField: UIView {
     }
     
     private func setupSendButton() {
-        sendButton.titleLabel?.font = Fonts.h10()
-        sendButton.setTitleColor(.white, for: .normal)
-        sendButton.setTitle("Send".localized, for: .normal)
+        sendButton.titleLabel?.font = Fonts.h8(weight: .bold)
+        sendButton.setTitleColor(.systemBlue, for: .normal)
```

Comment:
you work with xcode 11, `.systemBlue` does not exists in xcode 10.2 (the xcode that we are using right now)
please open this project with xcode 10.2 and use it to fix this problem because the app does not compile on xcode 10.2 right now

Reply:
Got it. I changed the color from .systemBlue to .blue